d2/src/main/java/com/linkedin/d2/discovery/util/D2Utils.java

```diff
@@ -17,6 +20,13 @@
 
 public class D2Utils
 {
+  // A set of system properties to be excluded as they are lengthy, not needed, etc.
+  private static final Set<String> SYSTEM_PROPS_TO_EXCLUDE = Stream.of(
+      "jdk.debug",
+      "line.separator",
+      "java.class.path",
+      "java.vm.inputarguments"
+  ).collect(Collectors.toSet());
```

Comment:
How about directly create a set
```suggestion
private static final Set<String> systemPropsToExclude = new HashSet<>(Arrays.asList(
    "jdk.debug",
    "line.separator",
    "java.class.path",
    "java.vm.inputarguments"
));
```

Reply:
what's the difference? why to prefer one over the other?

Reply:
It's just a tip, for the second one:
	•	Slightly more performant, especially for small sets.
	•	Less runtime overhead than streams.

Reply:
the diff is very negligible.  Since introduced in java 8,  Stream has been super commonly used to handle Set, List, Map entries, etc. It also saves a bit typing. (If Java 9 is available, I would use Set.of then, but with 8, I think Stream is fine).

Reply:
Yeah. I'm open to both method. It's just a tiny tip and yes, the diff is very negligible